src/tokenize.jl

```diff
@@ -1339,7 +1339,8 @@ end
 function simple_hash(str)
     ind = 1
     h = UInt64(0)
-    while ind <= length(str)
+    L = length(str)
+    while ind <= L
```

Comment:
```suggestion
    L = min(lastindex(str), MAX_KW_LENGTH)
    while ind <= L
```
is probably more correct and cheaper.